go/consensus/api/api.go

```diff
@@ -100,6 +103,27 @@ type Block struct {
 	Meta cbor.RawMessage `json:"meta"`
 }
 
+// Status is the current status overview.
+type Status struct {
+	// ConsensusVersion is the version of the consensus protocol that the node is using.
+	ConsensusVersion string `json:"consensus_version"`
```

Comment:
Maybe also include a `Backend` field that specifies the consensus backend identifier (as a string).

Reply:
Done! :)